src/highdicom/pr/__init__.py

```diff
@@ -0,0 +1,46 @@
+"""Package for creation of Presentation State instances."""
+
+from highdicom.pr.sop import (
+    GrayscaleSoftcopyPresentationState,
+    PseudoColorSoftcopyPresentationState,
+    ColorSoftcopyPresentationState,
+)
+from highdicom.pr.enum import (
+    AnnotationUnitsValues,
+    GraphicTypeValues,
+    PresentationLUTShapeValues,
+    TextJustificationValues,
+)
+from highdicom.pr.content import (
+    GraphicAnnotation,
+    GraphicGroup,
+    GraphicLayer,
+    GraphicObject,
+    SoftcopyVOILUT,
+    TextObject
+)
+
+
+SOP_CLASS_UIDS = {
+    '1.2.840.10008.5.1.4.1.1.11.1',  # Grayscale Softcopy Presentation State
+    '1.2.840.10008.5.1.4.1.1.11.2',  # Color Softcopy Presentation State
+    '1.2.840.10008.5.1.4.1.1.11.3',  # Pseudo Color Softcopy Presentation State
+}
```

Comment:
I initially added these dictionaries to allow one to check whether a given SOP Class UID represents a specific Modality. However, the packages no longer necessarily map one-to-one to modalities (e.g., "PM" is not a modality). It may also be fine to just include the UIDs of SOP classes that are actually implemented by the package.